### PR TITLE
Feature/jcs timedate func

### LINKED ISSUE #63: Add date (or day of week?) to the Weekly Forecast on Location page
- [ ] Display day of week on the weekly forecast data in Location
- [ ] Update timestamp on Location page in the top card to say the Date/Day of week next to the running time

### PR DESCRIPTION
**Pull request by JoshSevy**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Made display changes per team, did a little clean up in the components I was working on. 
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
- Add date conversion functions 
- Implemented into Location component and WeeklyForecast component
- Both components are rendering per teams feedback -- Thank You both !
- Moved functions being used in multiple components to conversion in helpers folder
- Fixed visibility conversion so its displaying correct miles
## Has This Been Tested?
- [ ] No
- [x] Yes
- Just visually but I do need to write tests for the functions in helper
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
-npm start ran through application to check work all just display
# Message for reviewer:
- Let me know if we need to add to any of these functions, they can be returned in any JSX format we could like
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Card/Card.tsx [hunk 1]
```diff
@@ -12,10 +12,11 @@ import daySunnyStorm from '../assets/10d.png'
 import sunnyStorms from '../assets/10n.png'
 import thunderstorm from '../assets/11d.png'
 import thunderRain from '../assets/11n.png'
-
 import snow from '../assets/13d.png'
 import mist from '../assets/50d.png'
 
+import { weatherIcon } from '../helpers/conversions'
```
Comment (leighlars) [inline src/Card/Card.tsx:18]: so clean!!!

File: src/Location/Location.tsx [hunk 2]
```diff
@@ -20,14 +20,10 @@ import thunderstorm from "../assets/11d.png";
 import snow from "../assets/13d.png";
 import mist from "../assets/50d.png";
 
-import { kelvinToFahren, degToDirection } from '../helpers/conversions'
-
-
+import { kelvinToFahren, degToDirection, weatherDtDisplay, weatherIcon } from '../helpers/conversions'
```
Comment (leighlars) [inline src/Location/Location.tsx:23]: i'm excited for this, perhaps I'll use this for styling too?

File: src/WeeklyForecast/WeeklyForecast.tsx [hunk 3]
```diff
@@ -47,41 +50,14 @@ const WeeklyForecast = (props: any) => {
...
-   } else if (icon === "50n" || icon === "50d") {
-    return <img src={mist} alt="Mist Icon" />;
-   }
-  };
-
-
-  const renderDailyWeather = props.weeklyWeather.map((day: any) => {
+  const renderDailyWeather = props.weeklyWeather.map((day: any, index: number) => {
     return (
+      (index !== 0) ?
```
Comment (leighlars) [inline src/WeeklyForecast/WeeklyForecast.tsx:55]: so good!

File: src/helpers/conversions.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```
Comment (ErinUntermeyer) [inline src/helpers/conversions.tsx:1]: Thanks for moving repeated functions here, it looks so much cleaner!

**Comment by ErinUntermeyer** [comment]
It looks awesome Josh, great job!

**Review by leighlars — APPROVED** [review]
Looks great!

**Comment by leighlars** [comment]
Great work

**Review by ErinUntermeyer — APPROVED** [review]

**Pull request merged**